### PR TITLE
Make License Validated

### PR DESCRIPTION
**Pull request by Xunzhuo**
Make Apache License Standard

Signed-off-by: Xunzhuo <mixdeers@gmail.com>

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright [2022] [The Envoy Project Authors]
```
Comment (arkodg) [inline LICENSE:189]:
```suggestion
   Copyright 2022 Envoyproxy Authors
```

Comment (arkodg) [reply LICENSE:189]: based on https://www.apache.org/licenses/LICENSE-2.0#apply

**Review by youngnick — APPROVED** [review]

File: LICENSE [hunk 2]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2022 The Envoyproxy Authors
```
Comment (daixiang0) [inline LICENSE:189]: I am curious if the year should be updated every year?

Comment (youngnick) [reply LICENSE:189]: It should get updated each time the file is touched.

**Review by alexgervais — APPROVED** [review]

File: LICENSE (hunk 2 above)
Comment (lizan) [reply LICENSE:189]: Read the paragraph above carefully, this part should be never changed: https://github.com/envoyproxy/envoy/pull/737#discussion_r112274371

**Review by lizan — CHANGES_REQUESTED** [review]
We should just follow https://github.com/cncf/foundation/blob/main/license-notices.md and add a NOTICE file instead.

**Comment by Xunzhuo** [comment]
thanks @lizan, I will update it later.

**Comment by skriss** [comment]
I'm going to close this out per https://github.com/envoyproxy/gateway/pull/134#discussion_r914232546, can open a new PR if there's another change to be made. Thanks @Xunzhuo.

**Pull request closed without merging**